tests/app/data_manually_created/0_star_wars.json

```diff
@@ -1,424 +1,514 @@
-                                    "label": "Yoda"
-                                  },
-                                     {
-                                    "value": "Anakin Skywalker",
-                                    "label": "Anakin Skywalker"
-                                  },
-                                  {
-                                    "value": "Rey",
-                                    "label": "Rey"
-                                  },
-                                     {
-                                    "value": "Obi-Wan Kenobi",
-                                    "label": "Obi-Wan Kenobi"
-                                  },
-                                  {
-                                    "value": "Qui-Gon Jinn",
-                                    "label": "Qui-Gon Jinn"
-                                  }
-                                ],
-                                "mandatory": true,
-                                "validation": {
+                                {
+                                    "answers": [
+                                        {
+                                            "display": {
+                                                "properties": {
+                                                    "columns": true
+                                                }
+                                            },
+                                            "guidance": "",
+                                            "id": "a5dc09e8-36f2-4bf4-97be-c9e6ca8cbe0d",
+                                            "label": "",
+                                            "mandatory": true,
+                                            "options": [
+                                                {
+                                                    "label": "Lion",
+                                                    "value": "Lion"
+                                                },
+                                                {
+                                                    "label": "Elephant",
+                                                    "value": "Elephant"
+                                                },
+                                                {
+                                                    "label": "Cow",
+                                                    "value": "Cow"
+                                                },
+                                                {
+                                                    "label": "Hippo",
+                                                    "value": "Hippo"
+                                                }
+                                            ],
+                                            "q_code": "4",
+                                            "type": "Radio",
+                                            "validation": {
```

Comment:
Is validation a required value here? I.e. even if there are no validation messages is this still required?